README.md

```diff
@@ -9,6 +9,7 @@ A curated list of awesome content created by Facebook Developer Circles communit
 - [Workshops](#workshops)
 - [Articles](#articles)
 - [Tools](#tools)
+- [First Contributions](#contributions)
```

Comment:
@Alquimista3301 I like your suggestions on adding more category to the list. Can you please amend the PR based on the suggestion from @aboullaite before we merge this? Thanks!